src/mono/netcore/sample/wasm/browser-profile/Makefile

```diff
@@ -0,0 +1,36 @@
+TOP=../../../../../..
+DOTNET=$(TOP)/dotnet.sh
+
+ifeq ($(V),)
+DOTNET_Q_ARGS=--nologo -v:q -consoleloggerparameters:NoSummary
+else
+DOTNET_Q_ARGS=--nologo
+DOTNET_MONO_LOG_LEVEL=--setenv=MONO_LOG_LEVEL=debug
+endif
+
+CONFIG?=Release
+
+build-aot-profile:
+	EMSDK_PATH=$(realpath $(TOP)/src/mono/wasm/emsdk) $(DOTNET) publish $(DOTNET_Q_ARGS) /p:EnableProfiler=true /p:WasmBuildNative=true /p:Configuration=$(CONFIG) /p:TargetArchitecture=wasm /p:TargetOS=Browser $(MSBUILD_ARGS) WasmSample.csproj
+
+run-aot-profiled:	
+ifeq ($(DOWNLOAD_DIR),)
+	$(error DOWNLOAD_DIR is not set)
+endif
+	cp $(DOWNLOAD_DIR)/data.aotprofile bin/$(CONFIG)/AppBundle
```

Comment:
Is this required? Or can we pass the original path to `/p:AOTProfilePath`?

Reply:
Yes, we can do that. Just doing that to keep everything in one place.